Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3563,11 +3569,19 @@ - (BOOL) containsValueForKey: (NSString *)key
         {
           hasValue = [self containsValueForKey: [XmlKeyMapTable objectForKey: key]];
         }
+      else if ([@"NSBackgroundColors" isEqualToString: key])
```

Comment:
Which value is it? "NSCollectionViewBackgroundColor" or "NSBackgroundColors"? It is highly unlikely that you will have to use a different value for these two methods.
Anyway, why didn't you just add this mapping to `XmlKeyMapTable`?

Reply:
```
    <key>NSBackgroundColors</key>
	<dict>
		<key>CF$UID</key>
		<integer>29</integer>
	</dict>
```
I believe it's NSBackgroundColors because that is what is in the nib representation of this.  Moved to the XmlKeyMapTable.

Reply:
```
 NO, representedObject = (null)
2022-09-24 07:54:11.108 NSCollectionViewLayout_test[120916:120916] Exception occurred while loading model: Can not determine type information for -[GSNamedColor count]
2022-09-24 07:54:11.108 NSCollectionViewLayout_test[120916:120916] Failed to load Xib
2022-09-24 07:54:11.108 NSCollectionViewLayout_test[120916:120916] Cannot load the main model file 'MainMenu'
```
Mapping it gets the above error, tracing down why, but I may re-instate the previous change temporarily.